webapp/templates/service-worker.js

```diff
@@ -26,8 +26,10 @@ self.addEventListener(
           return fetch(e.request)
             .then(r => {
               if (r.ok) {
-                const components = new RegExp('^/((bower_)?components|static)/');
-                if (components.test(e.request.url.toString())) {
+                const components = new RegExp('^/((bower_)?components|static|node_modules)/');
+                const wptEnvFlags = new RegExp('^/components/wpt-env-flags.js$');
```

Comment:
Can this be a simple string comparison instead of a regex?

Reply:
Done.